src/nav/infrastructure.yml

```diff
@@ -157,6 +157,8 @@ pages:
             path: /docs/infrastructure/prometheus-integrations/install-configure-openmetrics/install-update-or-uninstall-your-prometheus-openmetrics-integration
           - title: Configure Prometheus OpenMetrics integrations
             path: /docs/infrastructure/prometheus-integrations/install-configure-openmetrics/configure-prometheus-openmetrics-integrations
+          - title: Get Apache Flink integration for New Relic 
```

Comment:
for nav titles, we usually keep them pretty short. usual pattern would be 'Apache Flink integration'.